tests/module/animation/test_composition.py

```diff
@@ -169,3 +169,13 @@ def test_animationgroup_is_passing_remover_to_nested_animationgroups():
     assert sqr_animation.remover
     assert circ_animation.remover
     assert polygon_animation.remover
+
+
+def test_empty_animation_group_fails():
+    with pytest.raises(ValueError):
+        AnimationGroup().begin()
```

Comment:
Maybe check the text of the error?

Reply:
Maybe

Reply:
Actually i don't think we need to do that, the ValueError should be specific enough in that case. As long as it gives an error at that stage where good to go